PWGJE/Core/JetUtilities.h

```diff
@@ -138,10 +139,76 @@ std::tuple<std::vector<std::vector<int>>, std::vector<std::vector<int>>> MatchCl
   return std::make_tuple(matchIndexTrack, matchIndexCluster);
 }
 
+/**
```

Comment:
this should probably go in JetMatchingUtilities.h

Reply:
Hey @nzardosh ,

Should the old matching code also migrate to HetMatchingUtilities as well?

Reply:
Maybe its even better to make an `EMCALMatchingUtilities.h` or something like that? Since these are not jets. Unless you plan to use some of the features in the JetMatchingUtilities